<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/src/write2file.png</normaloff>:/src/write2file.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>900</width>
        <height>500</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>测试</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>180</width>
          <height>110</height>
         </rect>
        </property>
        <property name="title">
         <string>网络设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>端口号</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>IP</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="combo_port"/>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btn_net">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>连接主板</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="combo_ip"/>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>130</y>
          <width>180</width>
          <height>150</height>
         </rect>
        </property>
        <property name="title">
         <string>功能按键</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QPushButton" name="btn_ytest">
           <property name="text">
            <string>测试Y轴</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btn_ztest">
           <property name="text">
            <string>测试Z轴</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btn_switch">
           <property name="text">
            <string>开始测试</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_xtest">
           <property name="text">
            <string>测试X轴</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>5</y>
          <width>660</width>
          <height>430</height>
         </rect>
        </property>
        <property name="title">
         <string>信息</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="1">
          <widget class="QPushButton" name="btn_clear">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>清除</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QTextBrowser" name="textBrowser"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>打印</string>
       </attribute>
       <widget class="QGroupBox" name="gridGroupBox">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>130</y>
          <width>180</width>
          <height>125</height>
         </rect>
        </property>
        <property name="title">
         <string>MCU1-打印选项</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item row="4" column="0">
          <widget class="QCheckBox" name="cb_x">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>X轴</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QCheckBox" name="cb_z">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Z轴</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="cb_y">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Y轴</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QPushButton" name="btn_pOpen">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>设置对象</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="cb_net">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Net</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="cb_cpu">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>CPU</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="cb_r">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>R轴</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="cb_v">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>V轴</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="cb_p">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>POS</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QCheckBox" name="cb_AllSel">
           <property name="text">
            <string>全选</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>10</y>
          <width>180</width>
          <height>110</height>
         </rect>
        </property>
        <property name="title">
         <string>网络设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0">
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>端口号</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_">
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>IP</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="combo_port_p"/>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btn_net_p">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>连接主板</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="combo_ip_p"/>
         </item>
        </layout>
       </widget>
       <widget class="QTabWidget" name="tabWidget_Print">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>5</y>
          <width>660</width>
          <height>430</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabWidget_5Page1">
         <attribute name="title">
          <string>打印信息</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="3">
           <widget class="QPushButton" name="btn_clear_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>清除</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QTextBrowser" name="textBrowser_p"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btn_save">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>保存打印</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btn_pFresh">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>刷新打印</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="gridGroupBox">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>260</y>
          <width>180</width>
          <height>125</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="title">
         <string>MCU2-打印选项</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rb_PI">
           <property name="text">
            <string>入推杆</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="btn_pOpen_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>设置对象</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="rb_BI">
           <property name="text">
            <string>入皮带</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="rb_BO">
           <property name="text">
            <string>出皮带</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="rb_PO">
           <property name="text">
            <string>出推杆</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="rb_HR">
           <property name="text">
            <string>翻转盒</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <zorder>groupBox_2</zorder>
       <zorder>tabWidget_Print</zorder>
       <zorder>gridGroupBox</zorder>
       <zorder>gridGroupBox</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
